File: Modul-1/Python/PRAK108-2310817320007-ZahraNabila.py
print(f"Diketahui : ")

JumlahPutaran = int (input(f"Pak Dengklek mengelilingi taman = "))

jarak = int(input(f"Jarak tempuh Pak Dengklek = "))

r = jarak / JumlahPutaran
HasilKeliling = r / ( 2 * 3.14)

print(f"Jawaban : ")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah =  {HasilKeliling :.2f}")

File: Modul-1/C/PRAK108-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    float JumlahPutaran, jarak;

printf("Diketahui : \n");

printf("Pak Dengklek mengelilingi taman = ");
scanf("%f", &JumlahPutaran);

printf("Jarak tempuh Pak Dengklek = ");
scanf("%f", &jarak);

    float r = jarak / JumlahPutaran;
    float HasilKeliling = r / ( 2 * 3.14);

printf("Jawaban : \n");
printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah = %.2f\n", HasilKeliling);

    return 0;
}